app/src/test/java/com/metrics_board/app/controller/roll/ProjectControllerTest.java

```diff
@@ -172,11 +178,11 @@ public void testRequiredRequestBodyIsMissing() throws Exception {
     }
 
     @Test
-    public void testInternalServerError() throws Exception {
+    public void testInternalServerErrorWhenCreateProject() throws Exception {
         when(projectService.createProject(eq(OWNER_ID), any()))
                 .thenReturn(ProjectResponse.builder().build());
 
-        this.mockMvc.perform(post("/InvalidPath")
+        this.mockMvc.perform(post(INVALID_PATH)

```

Comment:
tests with `INVALID_PATH` not working correctly because you are not performing it against real web server wich respond with `404` on uknown url path.